typings/message.d.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference types="node" />
+
+import { CreateTypeDesc } from "./types";
```

Comment:
this is also an internal interface

Reply:
message is exported as property on the container instance. Hence, customers could do `rhea.message.modified()` and what would be the return type of this method. I found that it was the return type of the `CreateTypeDesc` interface. (I have defined the create() method on the TypeDesc as an interface)
```js
export interface CreateTypeDesc {
  (value?: any, code?: any, descriptor?: any): Typed;
  typecode?: number;
}
```
I found these methods to return that type. Hence I feel these are not internal per se.

Reply:
They are intended to be internal. The return value should be considered opaque. The reason being that it may change.